internal/pod_models_test.go

```diff
@@ -512,7 +512,7 @@ var _ = Describe("pod_models", func() {
 						"--log-path", "/var/log/fdb-trace-logs/monitor.log",
 					}))
 
-					Expect(mainContainer.Env).To(Equal([]corev1.EnvVar{
+					Expect(mainContainer.Env).To(ContainElements([]corev1.EnvVar{
```

Comment:
What was the reasoning behind this change? I believe if we want to change that it should be `ConsistOf` to make sure that exactly those env variables are present (without caring about the order).

Reply:
the new method will change the cap and cause `Equal()` checks to fail, I'll swap to `ConsistOf` no problem